velox/exec/Driver.h

```diff
@@ -126,8 +126,12 @@ enum class BlockingReason {
   kWaitForSplit,
   kWaitForExchange,
   kWaitForJoinBuild,
-  /// Build operator is blocked waiting for the probe operators to finish
-  /// probing before build the next hash table from the previously spilled data.
+  /// For a build operator, it is blocked waiting for the probe operators to
```

Comment:
Nice comment. It would be great to add similar comments to all enum values (in a separate PR).